R/deployApp.R

```diff
@@ -280,7 +280,60 @@ deployApp <- function(appDir = getwd(),
                      application$id), {
       bundlePath <- bundleApp(target$appName, appDir, appFiles,
                               appPrimaryDoc, assetTypeName, contentCategory)
-      bundle <- client$uploadApplication(application$id, bundlePath)
+      cat("doing upload", "\n")
+      if(!is.null(client$clientName) && client$clientName() == "lucid"){
```

Comment:
I think there is a function `isShinyapps()` to which you can pass `accountDetails`.